### PR TITLE
Ad 1 list of crypto currencies

### PR DESCRIPTION
**Pull request by agusss03**
I´ve finished the CriptoCurrency list.

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,7 +1,97 @@
...
+            />
+          </View>
+          <Text style={styles.btnAddCrypto}> + Add a Cryptocurrency</Text>
+        </View>
+      </SafeAreaView>
+    </>
+  );
 };
 
+const styles = StyleSheet.create({
```
Comment (thom-as-thom) [inline App.tsx]: Move styles to a styles.ts file and implement styled components

File: App.tsx [hunk 2]
```diff
@@ -1,7 +1,97 @@
...
+  container: {
+    backgroundColor: '#385775',
```
Comment (thom-as-thom) [inline App.tsx]: Usea theme.ts to define the colors

File: Cryptos.tsx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    price: number;
+    logo: any;
+    status: number;
+  };
+};
+
+const Item = ({crypto}: ItemProps) => (
+  <>
+    <View style={styles.wrapper}>
+      {/*Left side*/}
```
Comment (thom-as-thom) [inline Cryptos.tsx]: delete comments

File: App.tsx [hunk 4]
```diff
@@ -1,7 +1,97 @@
...
+    padding: 20,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    shadowColor: '#000',
+  },
+  textoCrypto: {
```
Comment (thom-as-thom) [inline App.tsx]: If there is a comonent that could be reused, as this one, usea more general name for reusability

File: Cryptos.tsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {View, Text, StyleSheet, Image} from 'react-native';
+
+type ItemProps = {
+  crypto: {
+    id: string;
+    name: string;
+    abr: string;
+    price: number;
+    logo: any;
```
Comment (thom-as-thom) [inline Cryptos.tsx]: dont use type any

File: App.tsx [hunk 6]
```diff
@@ -1,7 +1,97 @@
...
+    name: 'XRP',
+    abr: 'XRP',
+    price: 0.220568,
+    status: -2.47,
+  },
+];
+
 const App = () => {
-  return <></>;
+  return (
```
Comment (tomasdematos) [inline App.tsx]: use implicit return: 
() => (
<>
  ...
</>
)

File: App.tsx [hunk 7]
```diff
@@ -1,7 +1,97 @@
...
+            source={require('./assets/images/cara.jpg')}
+            style={styles.imagen}
+          />
+        </View>
+        <View style={styles.espacio} />
+        <View>
+          <View>
+            <FlatList
+              data={cryptos}
+              keyExtractor={item => item.id}
```
Comment (tomasdematos) [inline App.tsx]: use destructuring:

keyExtractor={{id} => id}

File: App.tsx [hunk 8]
```diff
@@ -1,7 +1,97 @@
...
-import {SafeAreaView, Text, View} from 'react-native';
+import {
+  Image,
+  SafeAreaView,
+  StyleSheet,
+  Text,
+  View,
+  FlatList,
+} from 'react-native';
+import Item from './Cryptos';
```
Comment (tomasdematos) [inline App.tsx]: Rename to Crypto, the file and Item

File: App.tsx (hunk 2 above)
Comment (tomasdematos) [reply App.tsx]: src/config/theme.ts


```
const theme = {
   colors: {
     black: "#000",
     .....
   }
}

export defautl theme;

```

File: App.tsx [hunk 9]
```diff
@@ -1,7 +1,97 @@
...
 };
 
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#385775',
+    padding: 20,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    shadowColor: '#000',
```
Comment (tomasdematos) [inline App.tsx]: replace all the colors with the theme

File: Cryptos.tsx [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+    status: number;
+  };
+};
+
+const Item = ({crypto}: ItemProps) => (
```
Comment (tomasdematos) [inline Cryptos.tsx]: Rename to Crypto, also rename the name of the file to Crypto

File: Cryptos.tsx [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+      <View style={styles.left}>
+        <Image source={crypto.logo} style={styles.logo} />
+        <View style={styles.titleView}>
+          <Text style={styles.titulo}>{crypto.name}</Text>
+          <Text>{crypto.abr}</Text>
+        </View>
+      </View>
+
+      {/*Right side*/}
+      <View style={styles.derecha}>
```
Comment (tomasdematos) [inline Cryptos.tsx]: write in english

File: components/pacientes.js [hunk 12]
```diff
@@ -0,0 +1,39 @@
+/* eslint-disable prettier/prettier */
```
Comment (tomasdematos) [inline components/pacientes.js]: You never use this file

**Review by tomasdematos — APPROVED** [review]

File: styles.ts [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+  font-weight: bold;
+`;
+export const ImageApp = styled(Image)`
+  width: 50px;
+  height: 50px;
+  border-radius: 50px;
+`;
+export const Space = styled(View)`
+  margin-top: 15px;
+`;
```
Comment (albaarg) [inline styles.ts:28]: Use spaces between functions

**Pull request merged**